java/mapping/supported.js

```diff
@@ -15,7 +15,8 @@
     'visualGridOptions',
     'variationGroupId',
     'hooks',
-    'timeout']
+    'timeout',
+	'pageId']
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
	"pageId"]
```